google/cloud/spanner/interval_test.cc

```diff
@@ -247,6 +253,63 @@ TEST(Interval, OutputStreaming) {
   EXPECT_EQ("1 year 2 months 3 days 04:05:06.123456789", os.str());
 }
 
+TEST(Interval, TimestampOperations) {
```

Comment:
nit: might want to separate this into multiple tests, so it is clear what each is testing, but fine to leave as is

Reply:
Thanks.  I think it is clear enough as to what is being tested.  And indeed, it may well be clearer testing these "inverse" operations right next to each other.